recipes/pyopenms/meta.yaml

```
@@ -44,13 +45,15 @@ requirements:
     - pandas
     - matplotlib-base
     - sysroot_linux-64 =2.17 # [linux]
+    - icu >=73.2,<74.0a0
   run: # run_exports in libopenms should take care of the libraries. But it does not because of bugs. So copy everthing again and again
-    - libopenms {{ version }}
+    - libopenms >=3.2.0
```

```suggestion
    - libopenms >={{ version }}
```
Can we make something like that?